File: frontend/src/components/Book/Book.js
import "./Book.scss";
import { NavLink } from "react-router-dom";
import { statuses } from "../Popups/AddModal";

export const Book = ({id, title, link, date, status, author}) => {
	const createdDate = new Date(date);

	return (
		<NavLink to={`/books/${id}`} className="book-wrapper" >
			<div className="image">
				<img src={link} alt="bookCover" className="bookCover"/>
			</div>

			<div className="info">
				<div className="title">{title}</div>

				<div className="additional">
					<div className="status">{statuses[status]} {status.toLowerCase()}</div>
					<div className="date">{createdDate.toDateString().slice(3)}</div>
				</div>

				<div className="author">{author}</div>
			</div>
		</NavLink>
	)
}

File: backend/LibraryApi/Models/Book.cs
namespace backend.Models;

public class Book
{
    public string id { get; set; } = null!;
    
    public string author { get; set; } = null!;
    
    public string title { get; set; } = null!;
    
    public string coverUrl { get; set; } = null!;
    
    public string status { get; set; } = null!;

    public string date { get; set; } = null;
}